config/locales/error_messages.en.yml

```diff
@@ -144,10 +144,14 @@ court:
 case_number:
   _seq: 50
   blank:
-    long: Enter a case number for example A20161234
-    short: Enter a case number
-    api: Enter a case number for example A20161234
+    long: Enter a case number or URN
+    short: Enter a case number or URN
+    api: Enter a case number of URN
   invalid:
+    long: The case number should be less than 21 characters and alphanumeric
```

Comment:
i would have one error message that covers both. long message can say something like
`The case number must be a case number (e.g. A20161234) or unique reference number ( less than 21 letters and numbers)`

Reply:
👍